articles/support-vector-machines/index.md

```diff
@@ -0,0 +1,443 @@
+ 2. **Hyperplane**: It's defined as the plane that separates the data into different classes.
+ 3. **Margin**: This is the path or gap between two lines separating different classes.
+ 
+
+The following diagram gives a visual representation of the support vector machine concept.
+
+ ![terms](support-vector-machines.jpg)
+
+***mathematical formulation***
+
+Given training vectors `𝑥`<sub>i</sub> `∈ R`<sup>n</sup>, `i=1,. . . , n`, in two classes, and a vector `𝑦 ∈ {1,−1},` our goal is to find `𝑤 ∈ R`<sup>`n`</sup>*** and `𝑏 ∈ R`  such that `w` is normal to the hyperplanes.  (*see figure below*)
+ 
+ ![enter image description here](vector.jpg)
+
+ `w.u>=c` ; ie. project `u` on the normal
+
+`c = -b`
+
+if `w.u + b >=0` ; then `u` is then classified as positive (decision rule).
+
+`w.x + b >=1`
+
+`w.x + b <=-1`
+
+introducing  `y`<sub>i</sub> such that `y`<sub>i</sub>`=+-1` for +ve positive samples and -ve for negative samples.
+
+*for the negative class* 
+
+*replacing y with `-1` in the equation we have* 
+
+`-(w.x + b) <=-1`
+
+*dividing both sides by `-1` we have*
+
+`w.x + b >= 1`
+
+and hence the equations turn out to be the same,
+
+`y`<sub>i</sub>`(w.x + b ) >=1`, for +ve samples
+
+`y' <sub>i</sub>`(w.x +b)>=1`, for -ve samples and so we can use one of them can be used as the decision function ie.
+
+`y(w.x + b)-1>=0`
+
+Introducing a constraint to the above equation so that whenever we have a sample on the gutter (support vectors), it should then give us 0.
+
+So we use:
+
+`y`<sub>i</sub>`(w.x + b)-1 = 0` as our contstraint.
+
+Now that we have both the decision function and its constraint we, therefore, need to find the width of the margin. We have:
+
+***width***  = `(x`<sub>i</sub> `- x ) = 2/`<sub>||`w`||</sub> as  our optimization problem.(*see fegure below*)
+
+![Width](Width.jpg)
+ 
+We now need to find its maximum.
+
+max <sup>`2`</sup>/<sub>`||w||`</sub>  or  min <sup>`1`</sup>/<sub>`2`</sub>`||w||`
+
+Our optimization problem with the constraint will now be,   
+
+`L` = <sup>1</sup>/<sub>2</sub> `||w||`<sup>2</sup> `-  Σ︁`<sub>i.....m</sub> `𝛼  [y(w.x + b)-1 ]`
+
+where `m` is the number of samples.
+
+Differentiating the above equation with respect to `w` we have: 
+  
+<sup>`dL`</sup>/<sub>`dw`</sub> ` = w -   Σ︁ 𝛼  y`<sub>i</sub> x<sub>i</sub> ` = 0` and so: `w = Σ︁`<sub>i...m</sub>` 𝛼 `<sub>i</sub> ` y`<sub>i</sub> `x`<sub>i</sub>
+  
+differentiating with respect to `b` we have: 
+
+<sup>`dL`</sup>/<sub>`db`</sub>` = - Σ︁ 𝛼 `<sub>i</sub>` y`<sub>i</sub>` = 0` so  `Σ︁`<sub>`i....m`</sub>` 𝛼  y `<sub>i</sub>` = 0`
+
+plugging back to the equation, we have:
+ 
+`L = `<sup>1</sup>/<sub>2</sub>`  ( Σ︁ 𝛼 `<sub>i</sub>` y `<sub>i</sub>` x`<sub>i</sub>`) (Σ︁ 𝛼 `<sub>j</sub>` y  `<sub>j</sub>` x`<sub>j</sub>`) - (Σ︁ 𝛼`<sub>i</sub>` y `<sub>i</sub>` x`<sub>i</sub>`) - (Σ︁𝛼 y `<sub>i</sub>`x`<sub>i</sub>`) -  Σ︁ 𝛼`<sub>i</sub>`  y `<sub>i</sub>` + Σ︁ 𝛼`<sub>i</sub>
```

Comment:
The equations are looking bad because you aren't using Latex for all equations. Consider using Latex for the entire equation, including the summation and alpha signs.

Reply:
$$ L = \sum_{\alpha} \frac{a}{b}^\alpha $$

Reply:
Preview this in vscode. Online Markdown editors don;t support Latex equations.